src/Data/Lens/At.purs

```diff
@@ -0,0 +1,45 @@
+module Data.Lens.At
+  ( At
+  , at
+  ) where
+
+import Prelude
+
+import Data.Identity (runIdentity, Identity(..))
+import Data.Map as M
+import Data.Maybe (Maybe(..), maybe)
+import Data.Set as S
+import Data.StrMap as SM
+import Data.Tuple
+
+import Data.Lens (LensP(), lens)
+import Data.Lens.Index (Index)
+
+class (Index m a b) <= At m a b where
```

Comment:
Ah, one thing that it does give you is the ability to implement the `Contains` type class without implementing `Index` or `At`. But I don't really see the point... Not to mention, `Contains` seems like a pretty silly type class; can you think of a time when you would use this rather than just `at`?